ios/RNSScreen.m

```diff
@@ -564,6 +564,26 @@ - (void)viewWillAppear:(BOOL)animated
     _shouldNotify = NO;
   }
 
+#if !TARGET_OS_TV
+  NSUInteger currentIndex = [self.navigationController.viewControllers indexOfObject:self];
+
+  if (@available(iOS 11.0, *) && currentIndex > 0) {
```

Comment:
```suggestion
  if (@available(iOS 11.0, *)) {
    if (currentIndex > 0) {
```
I'd separate these statements because the Obj-c compiler gets confused when you combine `available` with logic operators (because it shouldn't be checked conditionally - it's >=iOS 11 or it's not) and warns with

 ```
 @available does not guard availability here; use if (@available) instead
 ```